applications/IceTrays.md

```diff
@@ -0,0 +1,136 @@
+* # Crust Grant Proposal
+
+  * **Project Name:** IceTrays 
+  * **Team Name:** YYDS Team
+  * **Payment Address:** 0x3b61504b021669ef375217bBeEe9f6689792E4d4
+  * **Receivable Token:** USDT
+
+  > ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+  > ⚠️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
+
+  ### Overview
+
+  ​	IceTrays is a multitiered file storage API cluster built on Crust and IPFS,  and it's based on raft consensus. It also provide a mutable file system.
```

Comment:
y need consensus?

Reply:
I don't know what you mean... The cluster uses raft(https://github.com/hashicorp/raft).  It is a library for providing consensus.

Reply:
> https://github.com/hashicorp/raft

I know raft, I mean why this decentralized pin service need it?

Reply:
For example, I uploaded a file and wanted at least three IPFS nodes to store it to prevent it from being lost. The cluster leader selects three nodes to pin it according to the policy. At the same time, the cluster also records the file path information (each node maintains the same root CID). Related to crust, the cluster records all the crust order information and maintains the order renewal.

Reply:
For importent files, IPFS nodes is  also the hot storage cache for user to fetch. Crust network make sure it never been lost(Can crust node guarantee its quality of service? i'm not sure.). Maybe I dont want to pay cru token for files, I just pin them on IPFS nodes. It depends on user's choice.

Reply:
Correct.  The purpose is to make IPFS & CRUST service more stable and easy to use.